test/e2e/federatedresourcequota_test.go

```diff
@@ -38,7 +38,7 @@ import (
 	"github.com/karmada-io/karmada/test/helper"
 )
 
-var _ = ginkgo.Describe("FederatedResourceQuota auto-provision testing", func() {
+var _ = framework.SerialDescribe("TFederatedResourceQuota auto-provision testing", func() {
```

Comment:
what does the `TFederatedResourceQuota` mean?

Reply:
without this `T`, it will failed?

Reply:
typo, fixed